### DISCUSSION TITLE
How to prevent neo-tree to open on startup?

### DISCUSSION THREAD

**Discussion by vdegenne** [LazyVim/LazyVim#4439] [Q&A] [answered]
I always use `nvim .` to open current directory but the session will open with neo-tree on the left.
I usually use `<leader>ql` to retrieve last session or `<leader><leader>` to open the file I have in mind directly, in other words I tend to avoid using filesystem view as much as possible.

What's the easiest and most straightforward way to tell lazyvim not to open neo-tree on startup?

Thanks

**Comment by dpetka2001** [comment, marked as answer]
Just use plain `nvim` instead of `nvim .`??? Even without neo-tree, if you open Neovim with `nvim .`, it will open current directory with `Netrw` instead. This is normal Neovim behavior.

**Reply by vdegenne** [reply]
Omg.. sorry for that stupid question. I didn't know I could still have the drawer on `nvim` alone (if I ever needed it). Thanks for your answer @dpetka2001
